src/coreclr/jit/emitxarch.cpp

```diff
@@ -20309,6 +20341,15 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             }
             break;
 
+        case INS_push2:
+            // TODO-XArch-APX: to be verified.
```

Comment:
can we verify this?

Reply:
I checked internally and this is the guidance I received. https://uops.info/table.html doesn't have these new instructions yet. That's why I marked it as a `ToDo`